components/experimental/src/duration/format.rs

```diff
@@ -494,7 +427,7 @@ mod tests {
     #[test]
     fn test_digital_formatter() {
         let duration = Duration {
-            sign: DurationSign::Positive,
+            sign: DurationSign::Negative,
```

Comment:
Making the duration negative effectively tests code related to `sign_displayed` as well.